sql/core/src/test/scala/org/apache/spark/sql/TPCDSQuerySuite.scala

```diff
@@ -46,11 +47,7 @@ class TPCDSQuerySuite extends BenchmarkQueryTest with TPCDSSchema {
     "q81", "q82", "q83", "q84", "q85", "q86", "q87", "q88", "q89", "q90",
     "q91", "q92", "q93", "q94", "q95", "q96", "q97", "q98", "q99")
 
-  val sqlConfgs = Seq(
-    SQLConf.CBO_ENABLED.key -> "true",
-    SQLConf.PLAN_STATS_ENABLED.key -> "true",
-    SQLConf.JOIN_REORDER_ENABLED.key -> "true"
-  )
+  val sqlConfgs: Seq[(String, String)] = Nil
```

Comment:
BTW, this is not identical with the original status. I'm wondering if we don't need `SQLConf.CROSS_JOINS_ENABLED.key -> "true"` ? Previously, we had it.

Reply:
Yea, since it is enabled by default in 3.0+, I dropped it in the previous PR.